lib/ruby_lsp/ruby_lsp_rails/support/active_support_test_case_helper.rb

```diff
@@ -11,8 +11,6 @@ def extract_test_case_name(node)
         message_value = node.message
         return unless message_value == "test" || message_value == "it"
 
-        return unless node.arguments
-
```

Comment:
This is not in the direct scope of this PR, but I've kept it to a separate commit; it is used by the `DocumentSymbol` class. The line below already accomplishments what this does.